addusers.py

```diff
@@ -257,6 +246,30 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
+    
+    help_description = ("Add new users and projects to OpenStack using " 
+                        "data from Google Sheets.")
+    parser = argparse.ArgumentParser(description=help_description)
+    parser.add_argument('-c', '--config', 
+                        help='Specify configuration file.')
+
+    args = parser.parse_args()
+   
+    if args.config is not None:
+        CONFIG_FILE = set_config_file(args.config)
+    else:
+        CONFIG_FILE = set_config_file()
+
+    config = ConfigParser.ConfigParser()
+    config.read(CONFIG_FILE)
+
+    admin_user = config.get('auth', 'admin_user')
+    admin_pwd = config.get('auth', 'admin_pwd')
+    admin_project = config.get('auth', 'admin_project')
+    auth_url = config.get('auth', 'auth_url')
+    nova_version = config.get('nova', 'version')
```

Comment:
Yes, this is planned, I had wanted to think about the best way to set up the centralized config info.  Would you recommend defining a config object with these attributes?  It seems you did something like that in the setpass config.py, using oslo_config.